TestAssets/ProjectsWithTests/MultipleFrameworkProject/project.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {
+    "xunit": "2.1.0",
+    "dotnet-test-xunit": "1.0.0-rc2-150399-32"
```

Comment:
We'll need to get https://github.com/dotnet/coreclr.xunit/pull/6 in before we can merge this PR. It requires a build of dotnet-test-xunit that supports net451